recipe/build.py

```diff
@@ -34,7 +17,7 @@ def _main(args):
     # Copy cudatoolkit-dev-post-install.py to $PREFIX/bin
     src = recipe_dir_path / "cudatoolkit-dev-post-install.py"
     dst = prefix_bin_dir_path
-    copy_files(src, dst)
+    shutil.copy(src, dst)
```

Comment:
This might not end up with a `+x` permission anymore...? In #11, it just [worked](https://github.com/conda-forge/cudatoolkit-dev-feedstock/pull/11/commits/e22a336a438eff208dc16df222ca85a918aa116a) to switch the order of copying and chmod (but I'm running into the same ClobberWarnings for tqdm).

Reply:
I could be wrong here, but I don't think that this is what is causing the issues

```bash
ln: target `/home/conda/feedstock_root/build_artifacts/cudatoolkit-dev_1563094447926/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/bin/zstdmt' is not a directory
```

and not the clobberwarnings..

Looking closely, the build is succeeding on Osx but not on Linux... I'm still trying to track the real issue here, but I haven't figure it out yet.